### DISCUSSION TITLE
Device ID vs Query ID

### DISCUSSION THREAD

**Discussion by Zipties** [ESPresense/ESPresense#300] [Support]
I am a little confused about the purpose of querying the apple device IDs. My girlfriend's iPhone for example identifies as apple:1006:10-8, ad her watch as apple:1006:9-24. My watch is apple:1006:9-26. If I query them I can turn those into the model and build numbers but it seems like over-engineering something that is already unique.  Is querying the device only necessary if someone has multiple devices with the same ID?

**Comment by cbhiii** [comment]
I too have this question. I don't see anything in the limited documentation about it.

**Comment by DTTerastar** [comment]
Querying is only to help increase the uniqueness.  If your ids are good, do not use it.  When we query we miss advertisements for a second or 2 and it lowers battery life on the tracked device since it has to answer us.
